helpers/commit_helpers.rb

```diff
@@ -8,7 +8,7 @@ def commit_url(current_page)
     "https://github.com/alphagov/govuk-developer-docs/commit/#{commit_sha}"
   end
 
-  def last_updated(current_page)
+  def format_last_updated(current_page)
```

Comment:
In hindsight, agreed - have changed this to `last_updated` and `format_timestamp`.

I'd like to keep the formatting itself in the helper so that we can write tests for it. I anticipate it becoming more feature-rich later on, e.g. formatting in relative time ("3 days ago") if recent enough.